themes/OneDark.tmTheme

```diff
@@ -48,7 +50,7 @@
         <key>name</key>
         <string>Comment</string>
         <key>scope</key>
-        <string>comment, string.comment</string>
```

Comment:
I don't have a list, I inspect the editor and get them from CSS classes applied to elements.